jstp/src/main/java/com/metarhia/jstp/connection/ConnectionException.java

```diff
@@ -1,25 +1,49 @@
 package com.metarhia.jstp.connection;
 
 /**
- * Created by lundibundi on 8/23/16.
+ * {@link Connection} exception
  */
 public class ConnectionException extends Exception {
 
+  /**
```

Comment:
Maybe it'll be better to just refer (via @inheritDoc) to the super constructor in each of the constructors as they are only wrappers for them.